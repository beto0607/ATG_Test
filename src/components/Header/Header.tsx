import React from "react";
import logo from "../../assets/logo-placeholder.png";
const Header: React.FC = () => {
    return (
        <div>
            <img src={logo} alt="Logo" />
            <h3>Logo</h3>
        </div>
    );
};
export default Header;
