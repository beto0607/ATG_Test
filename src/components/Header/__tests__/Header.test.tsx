import React from 'react';
import { shallow } from 'enzyme';
import Header from '../Header';
import toJson from 'enzyme-to-json'
import { Navbar } from 'react-bootstrap'

describe('Header', () => {
    it('renders without crashing', () => {
        const component = shallow((<Header />));
        expect(toJson(component)).toMatchSnapshot()
    });
});