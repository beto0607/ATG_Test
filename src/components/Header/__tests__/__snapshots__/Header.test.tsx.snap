// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders without crashing 1`] = `
<Bootstrap(undefined)
  bg="dark"
>
  <NavbarBrand
    href="#"
  >
    <img
      alt="Logo"
      height="30"
      src="logo-placeholder.png"
    />
  </NavbarBrand>
</Bootstrap(undefined)>
`;
